src/assets/img/wawb-info-icon.svg

```diff
@@ -1,5 +1,5 @@
-<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
lots of unrelated image files changed here (not commenting on all of them) - please restore them.